test/user.test.js

```diff
@@ -401,6 +401,29 @@ describe('User', function() {
     var pass73Char = pass72Char + '3';
     var passTooLong = pass72Char + 'WXYZ1234';
 
+    it('rejects empty passwords creation', function(done) {
+      try {
+        User.create({email: 'b@c.com', password: ''}, function(err) {
+          if (err) return done(err);
+          done(new Error('User.create() should have thrown an error.'));
+        });
+      } catch (e) {
+        expect(e).to.match(/Invalid password/);
```

Comment:
Please verify `err.code`, not the error message. The message can change in the future (or when localized).